tests/test_style.py

```diff
@@ -502,13 +442,11 @@ def test_fetch_private_github_urls(tmp_path):
         "gh://andreoliwa/nitpick@develop/initial.toml",
         # Regular GitHub URL
         "https://github.com/andreoliwa/nitpick/blob/develop/initial.toml",
-        # Raw URL directly
-        "https://raw.githubusercontent.com/andreoliwa/nitpick/develop/initial.toml",
```

Comment:
This line was accidentally removed, right?

I added it back and pushed a commit.